src/Planet/Planet.jl

```diff
@@ -127,6 +132,16 @@ function year_anom end
 function orbit_semimaj end
 """ Total solar irradiance (W/m``^2``) """
 function TSI end
+""" J2000 epoch """
+function epoch end
+""" mean anomaly at the epoch (radians) """
+function M_epoch end
```

Comment:
Consider more verbose variable names for all, e.g., `mean_anom_0`, `obliquity`, `lon_perihel` etc.